tests/junit_common.py

```diff
@@ -384,8 +385,9 @@ def run_test(index: int, test: str, spec: TestSpec, backend: str,
 
     # run test
     skip_reason: str = suite_spec.check_pre_skip(test, spec, backend, nproc)
+    test_name_str: str = f'"{spec.name}", ' if spec.name else ''
```

Comment:
Should we require that all tests be named (many testing frameworks do), or perhaps automatically assign them names?

Reply:
Adding names would be a lot of changes to almost every test file. I'm not opposed, but its also another thing to manually change every time we make a new test. I could go either way

Reply:
It looks like every file with more than one test names them. We could assert that (if you have multiple tests they all need names), but I feel like the sed operation to add `name="default",` to all the unnamed ones in `t*.c` files is also fine. One thing I've considered is using libtest-mimic to support using `cargo test` to run these tests, and if I do that, I'll need to name them all (either explicitly or programmatically).

Reply:
I'm good with either approach